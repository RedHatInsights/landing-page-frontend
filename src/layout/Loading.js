import React from 'react';

import { Main } from '@red-hat-insights/insights-frontend-components/components/Main';
import { Spinner } from  '@red-hat-insights/insights-frontend-components/components/Spinner';

const Loading = () => (
    <Main>
        <Spinner centered/>
    </Main>
);

export default Loading;
