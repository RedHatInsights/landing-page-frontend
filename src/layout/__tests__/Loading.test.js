import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../Loading';

describe('Hero component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<Loading />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
