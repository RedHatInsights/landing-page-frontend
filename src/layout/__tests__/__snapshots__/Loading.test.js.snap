// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component should render correctly 1`] = `
<Connect(Main)>
  <Spinner
    centered={true}
  />
</Connect(Main)>
`;
